core/src/main/java/org/frankframework/configuration/digester/ValidateAttributeRule.java

```diff
@@ -161,10 +173,25 @@ private void checkDeprecationAndConfigurationWarning(String name, Method m) {
 			}
 
 			if (isDeprecated) {
-				addSuppressableWarning(msg, SuppressKeys.DEPRECATION_SUPPRESS_KEY);
+				addSuppressibleWarning(msg, SuppressKeys.DEPRECATION_SUPPRESS_KEY);
 			} else {
 				addLocalWarning(msg);
 			}
+			return;
+		}
+		// Check enum Configuration Warnings
+		if (!m.getParameters()[0].getType().isEnum())
```

Comment:
Als ik het goed begrijp is 'm' altijd een setter, en vermoed ik dat parameter met nummer [0] dus altijd de juiste / enige parameter is. Wellicht nog even verduidelijken in een comment?

Reply:
Kun je kijken of het zo duidelijker is @evandongen?